src/components/Modals/VerifyModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { View, Text, TouchableOpacity, Modal } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+import PropTypes from "prop-types";
+import { styles } from "./modalStyles";
+
+const VerifyModal = ({ onPress, showModal }) => {
+  return (
+    <Modal visible={showModal} transparent={true} animationType="fade">
+      <View style={styles.overlay}>
+        <View style={styles.modalContainer}>
+          <View style={styles.modal}>
+            <AntDesign name="checkcircle" size={50} color="#32CD32" />
+            <Text style={styles.modalText}>Digital work pass saved!</Text>
+            <TouchableOpacity
+              style={styles.modalButton}
+              onPress={() => onPress()}
```

Comment:
```suggestion
              onPress={onPress}
```
If you are not calling the method with additional param, you can just write it this way.. Usually you define function call if it looks like 
```
onPress = {() => onPress({foo: "bar"})}
```